playwright_test/page/chatwidget.page.js

```diff
@@ -0,0 +1,40 @@
+import { expect } from "@playwright/test";
+import { URL_DETAILS } from "../utils/Url";
+import { APP_ID } from "../utils/Constant";
+
+class ChatWidgetPage {
+    constructor(page) {
+        this.page = page;
+
+        //Locators
+        this.testRadioBtn = page.locator("#test");
+        this.appIDbox = page.getByPlaceholder(APP_ID.DEF_APP_ID);
+        this.launchWidgetBtn = page.getByRole("button", {name: "Launch Widget"});
+        this.kommunicateChatWidgetiframe = page.frameLocator('iframe[name="Kommunicate widget iframe"]');
```

Comment:
we can use id selector instead of name

Reply:
id is not present in project, have used xpath.